uwsgi.ini

```diff
@@ -0,0 +1,15 @@
+[uwsgi]
+project = river
+uid = uwsgi
+gid = uwsgi
+base = /srv/django
+
+chdir = %(base)
+module = %(project).wsgi
+master = True
+
+# TODO(vmttn): use wsgi protocol instead of http
```

Comment:
yes definitely, the reason we didn't do this before was because of nginx: I wouldn't find a way to forward  a http header using the `uwsgi_pass` option of nginx, but I don't thinbk that's relevant anymore.

Reply:
I could make use of `uwsgi_pass`, but I would rather think about it before.
The reason being that we would be unable to request the api without a nginx server. We could do this in another PR.

Reply:
👍